content/ko/docs/tutorials/kubernetes-basics/_index.html

```diff
@@ -45,25 +45,25 @@ <h2>쿠버네티스 기초 모듈</h2>
           <div class="row">
             <div class="col-md-4">
               <div class="thumbnail">
-                <a href="/docs/tutorials/kubernetes-basics/cluster-intro/"><img src="./public/images/module_01.svg?v=1469803628347" alt=""></a>
+                <a href="./create-cluster/cluster-intro/"><img src="/docs/tutorials/kubernetes-basics/public/images/module_01.svg?v=1469803628347" alt=""></a>
```

Comment:
@gochist 👋 I tried the full path of `/ko/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/` instead, and immediately saw why you updated to more relative paths: 
```
http://localhost:1313/ko/docs/tutorials/kubernetes-basics/ko/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro/
```

So thanks for these fixes. 👍